elements/pfe-modal/src/pfe-modal.js

```diff
@@ -96,6 +96,7 @@ class PfeModal extends PFElement {
 
     if (this.trigger) {
       this.trigger.addEventListener("click", this.open);
+      this.removeAttribute("hidden");
```

Comment:
@kylebuch8 Since this is a progressive enhancement feature, I suggested we remove the hidden attribute in the init function instead of the on click event.  Once the component upgrades, the shadow DOM manages the hidden nature of the modal panel and we don't want to risk hiding the trigger if it's used.  The hidden attribute on host would just be there to prevent a flash of unstyled content or to hide the content should JS fail (guidance would be to leave off the hidden attribute if content is critical to the page).